src/wiki/plugins/images/models.py

```diff
@@ -119,15 +119,35 @@ def on_image_revision_delete(instance, *args, **kwargs):
     if not instance.image:
         return
 
-    # Remove image file
+    # from attachments.models.py (line 166) :
+    # Remove file
+    path = instance.image.path.split("/")[:-1]
     instance.image.delete(save=False)
 
+    '''
```

Comment:
All of this code is commented out?

Reply:
it is part of the original code. it seems to check for the support of 'path' on the backend.
(path failed with ValueError when the image was deleted). 

mayebe add the check to the 'attachements' module too while I'm at it then?

Reply:
and add a test with that type of backend? (i do not no which backend applies for that)

Reply:
@superandrew added that part (for S3 support?). 

https://github.com/django-wiki/django-wiki/commit/9c7efff3b9310b6642fd4d7a5fc9456544ced8b8#diff-e54da208fb86cf66506a5f83e74bd916

So i think removing it is not such a good idea after all, and the attachments plugin should be taken care of also.
Ok?

Reply:
Nah, it's not a good idea to remove it, see my comment above.

Your change can probably be adapted into the current pattern?